docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -17,89 +19,101 @@ Extending Ansible with local modules and plugins offers lots of shortcuts such a
 * You do not have to add tests (though we recommend that you do!).
 
-To save a local module or plugin such that Ansible can find and use it, add the module or plugin in the appropriate directory (the directories are specified in later parts of this topic).
-
 .. contents::
    :local:
 
 .. _modules_vs_plugins:
 
 Modules and plugins: what is the difference?
 ============================================
-If you are looking to add local functionality to Ansible, you might wonder whether you need a module or a plugin. Here is a quick overview to help you decide between the two:
+If you are looking to add local functionality to Ansible, you might wonder whether you need a module or a plugin. Here is a quick overview to help you understand what you need:
 
-* Modules are reusable, standalone scripts that can be used by the Ansible API, the :command:`ansible` command, or the :command:`ansible-playbook` command. Modules provide a defined interface. Each module accepts arguments and returns information to Ansible by printing a JSON string to stdout before exiting. Modules execute on the target system (usually that means on a remote system) in separate processes.
-* :ref:`Plugins <plugins_lookup>` augment Ansible's core functionality and execute on the control node within the ``/usr/bin/ansible`` process. Plugins offer options and extensions for the core features of Ansible - transforming data, logging output, connecting to inventory, and more.
+* Modules are reusable, standalone scripts that can be used by the Ansible API, the :command:`ansible` command, or the :command:`ansible-playbook` command. Modules provide a defined interface. Each module accepts arguments and returns information to Ansible by printing a JSON string to stdout before exiting. Modules execute on the target system (usually that means on a remote system) in separate processes. Modules are technically plugins, but for historical reasons we do not usually talk about "module plugins".
+* :ref:`Plugins <working_with_plugins>` augment Ansible's core functionality and execute on the control node within the ``/usr/bin/ansible`` process. Plugins offer options and extensions for the core features of Ansible - transforming data, logging output, connecting to inventory, and more.
 
-.. _local_modules:
+.. _use_collections:
 
-Adding a module locally
-=======================
-Ansible automatically loads all executable files found in certain directories as modules.
+Adding modules and plugins in collections
+=========================================
 
-For local modules, use the name of the file as the module name: for example, if the module file is ``~/.ansible/plugins/modules/local_users.py``, use ``local_users`` as the module name.
+The best way to add local modules and plugins is to :ref:`create a collection <developing_collections>`. With a collection, you can use custom modules and plugins in any playbook or role. You can share your collection easily at any time through Ansible Galaxy.
```

Comment:
I think `local` shouldn't be here (for the above reasons):
```suggestion
The best way to add modules and plugins is to :ref:`create a collection <developing_collections>`. With a collection, you can use custom modules and plugins in any playbook or role. You can share your collection easily at any time through Ansible Galaxy.
```

Reply:
addressed in c3790e3887d1eb3d7b3437f5ec4794233f1ff03d